### PR TITLE
finished 'meet expectations' and removed pycache from repo

### PR DESCRIPTION
**Pull request by JaceLearnsPython**
# Met Expectations

I have delete and get working for the film view. I also removed the pycachce from my repo. 

I have also updated the UI a bit.

## UI
FIlms
image

Films URL Hover (A nice blue):
image

FIlm View
image

I may continue working on the other requests tomorrow depending on time.

Thank you!

### CODE REVIEW

File: ui/film.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+async function handleDelete(filmId) {
+  try {
+    const deleteResponse = await fetch(`/api/v1/film/${filmId}`, {
+      method: "DELETE",
+    });
+    if (deleteResponse.ok) {
+      console.log('Film deleted successfully!');
+      goToPage(); // Take me homeeeeeeeee To the placeeeeee I belong!!!!! West Virginia!!!!!
```
Comment (semmypurewal) [inline ui/film.jsx:16]: haha, i'm assuming this is referencing the John Denver song

File: ui/film.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+  // create the film view
+  const rootElt = document.getElementById("app");
+  const root = createRoot(rootElt);
+  root.render(
+    <div>
+      <h1>{film.title}</h1>
+      <li>Description: {film.description}</li>
+      <li>ID: {film.id}</li>
+      <button id="return-button" onClick={goToPage}>Return to Films</button>
```
Comment (semmypurewal) [inline ui/film.jsx:51]: This would be better as a link than a button (since it's not really performing an action, it's just a normal everyday link).

**Review by semmypurewal — APPROVED** [review]
Looks good!

File: ui/film.jsx (hunk 2 above)
Comment (JaceLearnsPython) [reply ui/film.jsx:51]: I agree. It started out as a link, but I did not like having a button and a link on the same page cause it looked bad in my opinion. I decided a button would make the page look cleaner and from the user's perspective it's like a "back" button.

**Pull request merged**